### PR TITLE
Task/pd 839 2fa global styles 2

### PR DESCRIPTION
**Pull request by catherine-jones**
- overrides for sign in h1 and h2
- overrides for h4
- adjust size of QR code
- overrides weight of label text
- overrides background colour of callout for QR code

### CODE REVIEW

File: app/views/users/shared_secrets/index.html.erb [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   <div class="qrcode">
     <p><strong>Scan the barcode directly into your Authenticator
       app.</strong></p>
-    <img src="/users/shared_secrets/code.png?nocache=<%= rand %>" />
+    <img alt="QR code" src="/users/shared_secrets/code.png?nocache=<%= rand %>" />
   </div>
-  <p><%= link_to 'Can\'t scan it?', code_users_shared_secrets_path %></p>
-  <p><%= link_to 'Next', new_users_shared_secret_path, role: :button %></p>
-  <p><%= button_to 'Generate a new code', users_shared_secret_path, method: :delete %></p>
+  <p class="link"><%= link_to 'Can\'t scan it?', code_users_shared_secrets_path %></p>
```
Comment (elisechant) [inline app/views/users/shared_secrets/index.html.erb]: assuming `<%= link_to` will render a link, in that case why do we have `link` class on `p`? should it not be on the link? if not, maybe we wrap the p's in a div and put a class on the parent instead in order to style the several `p` here?

File: client/src/login/styles/components/_sign-in-overrides.scss [hunk 2]
```diff
@@ -0,0 +1,99 @@
+article { //needed to over ride UI kit styles
```
Comment (elisechant) [inline client/src/login/styles/components/_sign-in-overrides.scss]: assuming we want to override the UI kit for all places not just login? maybe we can elevate these concerns to `src/styles/_common`. Why don't we can go through details today?

File: client/src/login/styles/components/_sign-in-overrides.scss [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        margin-top: 30px;
+        list-style-type: none;
+
+        li {
+          margin-bottom: 29px;
+        }
+      }
+    }
+
+    .button_to {
```
Comment (elisechant) [inline client/src/login/styles/components/_sign-in-overrides.scss]: any reason we have a random snake case here?

File: app/views/devise/shared/_links.html.erb [hunk 4]
```diff
@@ -1,11 +1,11 @@
 <%- if controller_name != 'sessions' %>
-  <%= link_to "Sign in", new_session_path(resource_name) %><br />
+  <li class="a"><%= link_to "Sign in", new_session_path(resource_name) %></li>
```
Comment (catherine-jones) [inline app/views/devise/shared/_links.html.erb]: that was just me debugging a loop with Josh yesterday

File: app/views/users/shared_secrets/index.html.erb [hunk 5]
```diff
@@ -1,15 +1,17 @@
...
-    <p><strong>Scan the barcode directly into your Authenticator
-      app.</strong></p>
-    <img src="/users/shared_secrets/code.png?nocache=<%= rand %>" />
+    <p><strong>Scan the QR code below into the app</strong></p>
+    <img alt="QR code" src="/users/shared_secrets/code.png?nocache=<%= rand %>" />
+  </div>
+  <div>
+    <div class="actions">
+      <%= button_to 'Generate a new code', users_shared_secret_path, method: :delete %>
+      <p><%= link_to 'Can\'t scan it?', code_users_shared_secrets_path %></p>
```
Comment (elisechant) [inline app/views/users/shared_secrets/index.html.erb]: why do links get wrapped in `p` but not `button`?

**Comment by elisechant** [comment]
@micapam 
can't run:

```
Started GET "/editor" for ::1 at 2016-11-11 09:42:20 +1100
Processing by EditorController#index as HTML
Completed 401 Unauthorized in 1ms (ActiveRecord: 0.0ms)


Started GET "/sign-in" for ::1 at 2016-11-11 09:42:20 +1100
Processing by Users::SessionsController#new as HTML
  Flipper::Adapters::ActiveRecord::Gate Load (0.5ms)  SELECT "flipper_gates".* FROM "flipper_gates" WHERE "flipper_gates"."feature_key" = $1  [["feature_key", "auth"]]
Filter chain halted as :check_auth_flag rendered or redirected
Completed 403 Forbidden in 2ms (ActiveRecord: 0.5ms)
```

File: app/views/users/shared_secrets/index.html.erb (hunk 5 above)
Comment (micapam) [inline app/views/users/shared_secrets/index.html.erb]: Not sure but there was excessive space around it yesterday and once we removed the `p` it looked better

File: client/src/login/styles/components/_sign-in-overrides.scss (hunk 2 above)
Comment (micapam) [inline client/src/login/styles/components/_sign-in-overrides.scss]: @elisechant Regarding this comment. I have no opinions on CSS structure so - is this a blocker in your view?

**Pull request merged**